extensions/ql-vscode/src/vscode-tests/no-workspace/query-history.test.ts

```diff
@@ -402,14 +402,29 @@ describe('query-history', () => {
       });
     });
 
-    it('should get the selection from the treeView when both selections are empty', async () => {
+    it('should get the selection from the treeView when both selections are empty', async function() {
+      this.timeout(999999999);
```

Comment:
Eeek...this should go away.

```suggestion
    it('should get the selection from the treeView when both selections are empty', async () => {
```